files/ca-setup.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
```

Comment:
I don't think I've ever seen us include an actual script in our docs repo that we reference. 

I'm guessing that this should probably be hosted somewhere else, but @vikram-redhat what are your thoughts on this?

Reply:
Thanks @bergerhoffer 
I did check with @vikram-redhat before adding this file. 
@vikram-redhat  can you confirm?

Reply:
This is ok.

Reply:
Great, thanks!